lib/methods/query.js

```diff
@@ -34,7 +34,17 @@ exports.query = function (Model, options, callback) {
 		whereQuery = this.translateWhereToQuery(where, values);
 	}
 
+	if (typeof options.order === 'string') {
+		options.order = options.order
+			.split(',')
+			.reduce(function (res, prop) {
+				res[prop] = 1;
+				return res;
+			}, {});
+	}
+
 	var order = Model.translateKeysForPayload(options.order);
+
```

Comment:
Oh and can you please remove the empty lines here and in L45? Then it's one big code block related to the order parameter ;)